site/docs/voice/bxml/startRecording.mdx

```diff
@@ -227,7 +227,9 @@ const speakSentenceStart = new SpeakSentence({
     voice: "bridget"
 });
 
-const startRecording = new StartRecording();
+const startRecording = new StartRecording({
```

Comment:
This is unrelated to the story, but I just noticed this example wasn't consistent